src/components/HelloWorld.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+// import PropTypes from 'prop-types';
+
+const GET_THINGS_REQUEST = 'GET_THINGS_REQUEST';
+export const GET_THINGS_SUCCESS = 'GET_THINGS_SUCCESS';
+
+export function getThingsSuccess(json) {
+  return {
+    type: GET_THINGS_SUCCESS,
+    json,
+  };
+}
+
+function getThings() {
+  console.log('getThings()Action!');
+  return (dispatch) => {
+    dispatch({ type: GET_THINGS_REQUEST });
+    return fetch('http://localhost:3000/greetings')
+      .then((response) => response.json())
+      .then((json) => dispatch(getThingsSuccess(json)))
+      .catch((error) => console.log(`Fetching Error ${error}`));
+  };
+}
```

Comment:
- [ ] OPTIONAL: I recommend placing get request to store and keeping the response in state 😉 
This way, you can access the state from any file and you application can scale!